test/bigtest/interactors/queryFileUploaderInteractor.js

```diff
@@ -24,4 +25,9 @@ import CalloutInteractor from '@folio/stripes-components/lib/Callout/tests/inter
   callout = new CalloutInteractor();
 }
 
+class FileExtensionsModal extends ConfirmationModalInteractor {
+  header = new Interactor('[data-test-file-extension-modal-header]');
+}
+
+export const fileExtensionModal = new FileExtensionsModal('#file-extension-modal');
```

Comment:
As far as I understand fileExtensionModal can be added to QueryFileUploaderInteractor or FileExtensionsModal should be moved to the separated file. @Yuliia-Dovhal @VictorSoroka1 what do you think about it?

Reply:
Sound good to add as part of `QueryFileUploaderInteractor`.